protocol/messenger_handler.go

```diff
@@ -2282,20 +2277,17 @@ func (m *Messenger) handleChatMessage(state *ReceivedMessageState, forceSeen boo
 	if receivedMessage.ContentType == protobuf.ChatMessage_COMMUNITY {
 		m.logger.Debug("Handling community content type")
 
-		communityResponse, err := m.handleWrappedCommunityDescriptionMessage(receivedMessage.GetCommunity(), common.ShardFromProtobuff(receivedMessage.Shard))
+		signer, description, err := communities.UnwrapCommunityDescriptionMessage(receivedMessage.GetCommunity())
```

Comment:
Do we still handle the shard somewhere?

Reply:
Good point :thinking:

Reply:
https://github.com/status-im/status-go/blob/3805662a1834dd65c2b317156b2ce4e1ee2a3950/protocol/messenger_communities.go#L2859

It seems we don't use shard when we handle community description :man_shrugging: - nil is hardcoded

Reply:
Interesting. @richard-ramos do we need the shard?

Reply:
I have a TODO for that: https://github.com/status-im/status-go/blob/develop/protocol/communities/manager.go#L464  but i have done some dogfooding past week and I think the shard ain't necessary in here. I can send a PR to remove it completely from handleWrappedCommunityDescriptionMessage

Reply:
Thanks @richard-ramos , I removed `handleWrappedCommunityDescriptionMessage` completely with this PR, there is no need to address that.